builder/gen-dockerfile/tests/test_data/values_on_both/app.yaml

```diff
@@ -0,0 +1,24 @@
+env: flex
+runtime: php
+
+runtime_config:
+  whitelist_functions: exec
+  front_controller_file: app.php
+  nginx_conf_http_include: files/nginx-http.conf
+  nginx_conf_include: files/nginx-app.conf
+  nginx_conf_override: files/nginx.conf
+  php_fpm_conf_override: files/php-fpm.conf
+  php_ini_override: files/php.ini
+  supervisord_conf_addition: files/additional-supervisord.conf
+  supervisord_conf_override: files/supervisord.conf
+
+env_variables:
```

Comment:
At runtime, these will be set by App Engine and overwrite the ones set in the runtime config. Is this something we want to warn or even fail the build for?